src/sync/pull-request.ts

```diff
@@ -74,14 +82,60 @@ const doGetPullRequestTaskInParallel = (
 	numberOfPagesToFetchInParallel,
 	pageSizeAwareCursor,
 	(pageCursor) =>
-		doGetPullRequestTask(
+		doGetPullRequestTaskRest(
 			logger, gitHubInstallationClient, jiraHost, repository,
 			pageCursor,
 			messagePayload
 		)
 );
 
+const emitStats = (jiraHost: string, startTime: number, requestType: string) => {
+	statsd.timing(
+		metricHttpRequest.syncPullRequest,
+		Date.now() - startTime,
+		1,
+		{ requestType },
+		{ jiraHost }
+	);
+};
+
 const doGetPullRequestTask = async (
```

Comment:
For added clarity, can we call these `doGetPullRequestTaskGraphQL` and `doGetPullRequestTaskRest`? Or even `fetchPullRequestsGraphQL` and `fetchPullRequestsREST`? I've always hated the bad grammar of do get... :cringe:

Reply:
You ignoring me Mr Kay?

Reply:
Added graphql on the end, dropped the do, they are child processes of the parent getPullRequestTask function so kept them aligned... if this variant works though we should be able to crush them all into a simple function a a simple name